tests/scripts/expect/cli-misc.exp

```diff
@@ -111,6 +111,11 @@ expect "Done"
 
 send "version\n"
 expect "Done"
+send "version api\n"
+expect "28"
```

Comment:
this will fail when API version is increased to 29.
```suggestion
expect -re {(\d+)}
```

Reply:
I think we should also change here when the API version is increased.

Reply:
Would it be too strict? Having to change two places for API version change also seems redundant. Thoughts?

Reply:
Indeed, that's somewhat redundant. I was thinking about checking the correctness of the CLI function implementation, but with a function simple as that it seems unnecessary.